x-pack/plugins/security_solution/public/detections/mitre/get_mitre_threat_component.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Threats } from '@kbn/securitysolution-io-ts-alerting-types';
+import type { SearchHit } from '../../../common/search_strategy';
+import { buildThreatDescription } from '../components/rules/description_step/helpers';
+
+export const getMitreComponentParts = (searchHit?: SearchHit) => {
```

Comment:
This replaces the unnecessary code I had in x-pack/plugins/security_solution/public/detections/pages/alert_details/tabs/summary/get_mitre_threat_component.ts previously